sql/telemetry_derived/clients_daily_histogram_aggregates_v1/query.sql

```diff
@@ -93,9 +93,29 @@ WITH filtered AS (
     AND normalized_channel IN ("release", "beta", "nightly")
     AND client_id IS NOT NULL
 ),
+sampled_data AS (
+  SELECT
+    *
+  FROM
+    filtered
+  WHERE
+    channel IN ("nightly", "beta")
+    OR (channel = "release" AND os != "Windows")
+  UNION ALL
+  SELECT
+    *
+  FROM
+    filtered
+  WHERE
+    channel = 'release'
+    AND os = 'Windows'
+    AND sample_id >= @min_sample_id
```

Comment:
The airflow jobs will need to be modified to provide default for these new variables.

Reply:
yep. will write that up when we make a decision on size.